iceoryx_examples/icecallback_on_c/README.md

```diff
@@ -1,4 +1,4 @@
-# [NOT YET IMPLEMENTED] icecallback_on_c - Using the WaitSet to implement callbacks
+# icecallback_on_c - Using the WaitSet to implement callbacks
```

Comment:
Can you name it "icecallback in C" without the underscores to be consistent with "icedelivery in C".

Reply:
done